### PR TITLE
Add Refresh container

### PR DESCRIPTION
**Pull request by jimmyxian**
Fix: #748 
Signed-off-by: Xian Chaobo <xianchaobo@huawei.com>

### CODE REVIEW

File: api/router.go [hunk 1]
```diff
@@ -52,14 +52,14 @@ var routes = map[string]map[string]handler{
 		"/images/{name:.*}/push":        proxyImage,
 		"/images/{name:.*}/tag":         proxyImageAndForceRefresh,
 		"/containers/create":            postContainersCreate,
-		"/containers/{name:.*}/kill":    proxyContainer,
-		"/containers/{name:.*}/pause":   proxyContainer,
-		"/containers/{name:.*}/unpause": proxyContainer,
+		"/containers/{name:.*}/kill":    proxyContainerAndForceRefresh,
```
Comment (aluzzardi) [inline api/router.go]: @vieux Could you look at the routes and make sure we didn't miss anything?

For instance, I'm wondering if `/containers/{name:.*}/export` needs a refresh.

Also, what about `/exec/{execid:.*}/start`?

File: test/integration/api/cp.bats [hunk 2]
```diff
@@ -19,8 +19,7 @@ function teardown() {
 	docker_swarm run -d --name test_container busybox sleep 500
 
 	# make sure container is up
-	# FIXME(#748): Retry required because of race condition.
-	retry 5 0.5 eval "[ -n $(docker_swarm ps -q --filter=name=test_container --filter=status=running) ]"
+	[ -n $(docker_swarm ps -q --filter=name=test_container --filter=status=running) ]
```
Comment (aluzzardi) [inline test/integration/api/cp.bats]: @jimmyxian Could you grep on the code and make sure there are no more `#748` left?

Comment (jimmyxian) [inline test/integration/api/cp.bats]: @aluzzardi Aleady greped, no more left

**Comment by jimmyxian** [comment]
@aluzzardi @vieux 
I have already fixed the `proxyAsync`. and use `proxyAsync` in `proxyContainerAndForceRefresh`
Check again, thanks.

**Comment by aluzzardi** [comment]
LGTM

**Pull request merged**